src/azure-cli/azure/cli/command_modules/vm/_help.py

```diff
@@ -2758,6 +2758,7 @@
 helps['vmss update'] = """
 type: command
 short-summary: Update a VMSS.
+long-summary: To update a global scale set property, you must update the property in the scale set model.Remember to execute the `az vmss update-instances -g 'Your resource group name' -n 'Your VMSS name' --instance-ids '*' ` command after using this command to make the change take effect. If you don't set the automatic modes for the upgrade policy.
```

Comment:
Put it to short summary. Long summary is not shown in references.

Reply:
OK，I've tried to write it in the short summary.